### PR TITLE
New README file

### LINKED ISSUE #18: Update README.md

### PR DESCRIPTION
**Pull request by burningtree**
There is a draft of new README file for newcomers.

Close #18

### CODE REVIEW

**Comment by burningtree** [comment]
It will be coot to append some section about how to donate - @coinmandeer

File: README.md [hunk 1]
```diff
@@ -1,35 +1,53 @@
...
 
-We collecting informations about:
+[Homepage](https://data.opencrypto.io) | [Data Explorer](http://explorer.opencrypto.io) | [Schema](https://schema.opencrypto.io)
+
+## Features
+* 👥 **Open & Transparent** - Everybody can review changes in [Commit history](../../commits/master), introduce new edits through [Pull Request](../../pulls) or submit a new [Issue](https://github.com/opencrypto-io/data/issues/new) or help with some [existing one](../../issues).
+* 📖 **Structured** - Whole database is defined in [JSON Schema](https://json-schema.org/) standard ([schema](https://schema.opencrypto.io)). See [Structure](#structure) section below.
+* 🔗 **Consistent** - Each change is tested agains schema (typed), so its not possible to release corrupted data or properties. Changes to [schema](https://schema.opencrypto.io) occurs rare and will be backward compatible. 
+* 🌱 **Platform-agnostic** - Database source is just plain-text [YAML](http://yaml.org/) files and because of that isn't dependent on any database software, platform or specific environment. You can just use [jq](https://stedolan.github.io/jq/) to browse it.
```
Comment (cryptomari0) [inline README.md]: because of that **it** isn't

- I think, sounds more correct to me

File: README.md [hunk 2]
```diff
@@ -1,35 +1,53 @@
...
+* 🔋 **Compact & Embeddable** - Automated builds produce a single-file version of full database ([data.json](https://data.opencrypto.io/data.json)), which can be easily downloaded/updated/used in any environment (base64 encoded icons and logos included!). See [Builds](#builds) section for more information.
+* ⏰ **Up-to-date** - Updates are pushed regularly, Pull Requests are resolved as soon as possible. Automated checks helps to detect outdated links and identify other issues. 
+* 🎁 **Free for forever**! - Data is licensed under [ODC Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/summary/).
```
Comment (cryptomari0) [inline README.md]: **Forever free** or **Free forever**

- more often I have found Forever free

File: README.md [hunk 3]
```diff
@@ -1,35 +1,53 @@
...
+
+
+## Structure
+
+All models included in this dataset is specified by [JSON Schema](http://json-schema.org/) vocabulary in separate repository [opencrypto-io/schema](https://github.com/opencrypto-io/schema) Continue to [schema.opencrypto.io](https://schema.opencrypto.io/) for inspect/explore schema and related informations.
```
Comment (cryptomari0) [inline README.md]: All **included models** in this dataset **are** specified

**Comment by burningtree** [comment]
@cryptomari0 Good work 👍 Please send me addresses and I will add this "Donate" section. Or you can add it by yourself.

**Pull request merged**